public/locales/en/profile.json

```diff
@@ -0,0 +1,18 @@
+{
+  "aboutHeader": "About {{firstName}}","user": "User",
```

Comment:
Super nit formatting thing, but I think the `"user": "User"` could go on the next line